### PR TITLE
upd(scss) use grey palette for disabled state on switch component

### PR DESCRIPTION
**Pull request by MarcFairbrother**
## Description

- Uses grey palette for disabled state on both checked and unchecked switch component
- Fixes inconsistencies between different HTML syntaxes (with wrapping `<label>` or `<div>`)
- Adds story for switch components

-----

### CODE REVIEW

**Review by RipripKipkip — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1483

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1483

**Review by AlexiaAlarza — APPROVED** [review]

File: stories/scss/form/switch/switch.stories.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<div class="grid">
+	<div class="grid-xxxs4">
+		<div class="form-group">
+			<label class="switch" [ngClass]="{'mod-small': isSmall, 'mod-inline': isInline}">
+				<input class="switch-input" id="switch1" type="checkbox" [checked]="isChecked" [disabled]="isDisabled"/>
```
Comment (vvalentin-lucca) [inline stories/scss/form/switch/switch.stories.html:5]:
```suggestion
				<input class="switch-input" id="switch1" type="checkbox" [checked]="isChecked" [disabled]="isDisabled ? 'disabled': null"/>
```

**Review by vvalentin-lucca — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1483

**Pull request merged**